changelog/is_input_range_element.dd

```diff
@@ -0,0 +1,5 @@
+isInputRange now takes an optional 2nd template parameter that
```

Comment:
`changelog/is_input_range_element.dd: After the title line an empty, separating line is expected`

So 
```
InputRange now takes an optional 2nd template parameter that defaults to void.

If not void, it only evaluates to true if the
 range's element type is convertible to passed-in parameter. For
 instance, `isInputRange!(int[], int)` is true, but
 `isInputRange!(int[], string)` is false. 
```
is the format you want